src/hotkey/counter_table.c

```diff
@@ -0,0 +1,242 @@
+#include "counter_table.h"
```

Comment:
Sorry for the possible bike-shedding on naming. But counter_table doesn't seem the most intuitive to me. May I propose "key-count map" or "kc_map" as an alternative?